### PR TITLE
Translated App Card in French

### LINKED ISSUE #4187: [localization] Translate App Card in French
### Is your proposal related to a problem?

Add translations for the App Card in French

### Describe the solution you'd like

The strings need to be translated:

Screenshot 2022-09-30 at 3 43 04 PM

**Here's the guide to contributing to [localization](https://docs.tooljet.com/docs/contributing-guide/l10n)**

### PR DESCRIPTION
**Pull request by vedant-z**
Fixes #4187

### CODE REVIEW

**Comment by vedant-z** [comment]
@withshubh Please review

**Comment by withshubh** [comment]
Please update the appropriate title and description.
We will review the PR soon.

File: frontend/assets/translations/fr.json [hunk 1]
```diff
@@ -339,19 +339,19 @@
...
-            "exportApp":"Export app",
-            "cloneApp":"Clone app",
-            "launch":"Launch",
-            "maintenance":"Maintenance",
-            "noDeployedVersion":"App does not have a deployed version",
-            "openInAppViewer":"Open in app viewer",
-            "removeFromFolder":"Remove from folder"
+            "changeIcon":"Icône de modification",
+            "addToFolder":"Ajouter au dossier",
+            "move":"Bouger",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:344]:
```suggestion
            "move":"Déplacer",
```

File: frontend/assets/translations/fr.json [hunk 2]
```diff
@@ -339,19 +339,19 @@
...
+            "to":"À",
+            "selectFolder":"Sélectionner un dossier",
+            "deleteApp":"Supprimer l'application",
+            "exportApp":"Exporter l'application",
+            "cloneApp":"Cloner l'application",
+            "launch":"Lancer",
+            "maintenance":"Entretien",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:351]:
```suggestion
            "maintenance":"Maintenance",
```

**Comment by vedant-z** [comment]
@withshubh Are there any more changes to be made ?

**Review by withshubh — APPROVED** [review]

**Comment by vedant-z** [comment]
@withshubh Sir

Is the Hacktoberfest swag T-shirt offered for each Hacktoberfest PR merged or is there one T-Shirt for all the PRs merged

**Pull request merged**